bastion/task/workflow/notion_to_s3.go

```diff
@@ -228,15 +228,31 @@ func (n *notionToS3Workflow) createAndUploadContentJson(content interface{}) err
 }
 
 func (n *notionToS3Workflow) getContent(paragraph *notion.Paragraph) (string, error) {
-	if paragraph == nil || paragraph.RichTexts == nil || len(paragraph.RichTexts) > 1 {
+	if paragraph == nil || paragraph.RichTexts == nil {
 		return "", fmt.Errorf("invalid paragraph")
 	}
 
 	if len(paragraph.RichTexts) == 0 {
 		return "\n", nil
 	}
 
-	return paragraph.RichTexts[0].PlainText + "\n", nil
+	if len(paragraph.RichTexts) == 1 {
+		return paragraph.RichTexts[0].PlainText + "\n", nil
+	}
+
+	var text string
+	for _, richText := range paragraph.RichTexts {
```

Comment:
Are you trying to build a translator of richText to markdown?  This part can be a lot of work. Please consider separate it out and use/build a library for it.